prow/entrypoint/options.go

```diff
@@ -65,6 +72,7 @@ func (o *Options) LoadConfig(config string) error {
 // BindOptions binds flags to options
 func (o *Options) BindOptions(flags *flag.FlagSet) {
 	flags.IntVar(&o.TimeoutMinutes, "timeout", DefaultTimeoutMinutes, "Timeout for the test command.")
+	flags.IntVar(&o.GracePeriodSeconds, "grace-period", DefaultGracePeriodSeconds, "Grace period after timeout for the test command.")
```

Comment:
golang supports duration flags, is there a reason we do not want to type `GracePeriod` this way? https://golang.org/pkg/flag/#Duration

This is how `kubetest` works: https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L179

Reply:
Impossible to serialize JSON to `time.Duration` directly... so would need internal and external config fields, which I am really not a fan of.